api/domain_handlers.go

```diff
@@ -381,3 +381,55 @@ func (api API) Remove(w http.ResponseWriter, r *http.Request) {
 
 	writeJSONOrBust(w, issues)
 }
+
+// Example: GET /ineligible?
+func (api API) Ineligible(w http.ResponseWriter, r *http.Request) {
```

Comment:
Add a comment explaining what this endpoint/function does.

This endpoint/function can be renamed to be clearer about what it does.

(Eventually this needs to be registered in server.go, but it's good to leave that out of this PR, as there's additional work that we probably want to do before the server is configured to call this function.)